### PR TITLE
NEXT-18240 - App feedback requests (tab and modal) are signed

### PR DESCRIPTION
**Pull request by fschmtt**
If an app provides feedback to the Administration after triggering an action button, the requests sent out for the tab and modal will have additional meta-information as query parameters.

To allow the app server to verify the authenticity of the incoming requests, the query part of the request is signed.

### CODE REVIEW

File: guides/plugins/apps/administration/add-custom-action-button.md [hunk 1]
```diff
@@ -74,6 +75,18 @@ Again you can verify the authenticity of the incoming request, like with [webhoo
 This feature was added in Shopware 6.4.3.0, previous versions will ignore the response content.
 {% endhint %}
 
+{% hint style="info" %}
+Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) will have the following additional query parameters:
+* shop-id
```
Comment (fschmtt) [inline guides/plugins/apps/administration/add-custom-action-button.md:80]: I hope the hint boxes allow listings, I cannot generate a preview 👀

File: guides/plugins/apps/administration/add-custom-action-button.md [hunk 2]
```diff
@@ -36,7 +36,8 @@ An action button must have the following attributes:
 * `entity`: Here you define which entity you're working on.
 * `view`: `detail`or `list`;  to set the view the button should be added to. Currently, you can choose between detail and listing view.
 
-When the user clicks on the action button your app receives a request similar to the one generated by a [webhook](../app-base-guide.md#webhooks). The main difference is that it contains the name of the entity and an array of ids that the user selected \(or an array containing just a single id in case the detail page\).
+When the user clicks on the action button your app receives a request similar to the one generated by a [webhook](../app-base-guide.md#webhooks).
+The main difference is that it contains the name of the entity and an array of ids that the user selected \(or an array containing just a single id in case the detail page\).
```
Comment (SebastianFranze) [inline guides/plugins/apps/administration/add-custom-action-button.md:40]:
```suggestion
The main difference is that it contains the name of the entity and an array of ids that the user selected \(or an array containing just a single id if the request was sent from a detail page\).
```

**Review by SebastianFranze — CHANGES_REQUESTED** [review]

File: guides/plugins/apps/administration/add-custom-action-button.md [hunk 3]
```diff
@@ -74,6 +75,18 @@ Again you can verify the authenticity of the incoming request, like with [webhoo
 This feature was added in Shopware 6.4.3.0, previous versions will ignore the response content.
 {% endhint %}
 
+{% hint style="info" %}
+Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) will have the following additional query parameters:
```
Comment (msestendrup) [inline guides/plugins/apps/administration/add-custom-action-button.md:79]:
```suggestion
Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) have the following additional query parameters:
```

Comment (xPand4B) [reply guides/plugins/apps/administration/add-custom-action-button.md:79]: Shouldn't it be "[...] and custom modal **HAS** the following additional [...]"?

**Review by SebastianFranze — APPROVED** [review]

**Review by keulinho — APPROVED** [review]

**Review by msestendrup — APPROVED** [review]

**Pull request merged**